uk.co.compucorp.civicrm.hrleaveandabsences/views/my-leave/components/my-leave-calendar.html

```diff
@@ -0,0 +1,386 @@
+            <span class="chr_leave-calendar__legend__label">Requested Leave AM</span>
+          </div>
+          <div class="chr_leave-calendar__legend__entry">
+            <div class="chr_leave-calendar__item chr_leave-calendar__item--requested chr_leave-calendar__item--type--leave"><span>PM</span></div>
+            <span class="chr_leave-calendar__legend__label">Requested Leave PM</span>
+          </div>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-sm-3">
+          <div class="chr_leave-calendar__legend__entry">
+            <div class="chr_leave-calendar__item chr_leave-calendar__item--type--toil"></div>
+            <span class="chr_leave-calendar__legend__label">TOIL</span>
+          </div>
+        </div>
+        <div class="col-sm-3">
+          <div class="chr_leave-calendar__legend__entry">
+            <div class="chr_leave-calendar__item chr_leave-calendar__item--type--toil"><span>AM</span></div>
+            <span class="chr_leave-calendar__legend__label">TOIL AM Only</span>
+          </div>
+          <div class="chr_leave-calendar__legend__entry">
+            <div class="chr_leave-calendar__item chr_leave-calendar__item--type--toil"><span>PM</span></div>
+            <span class="chr_leave-calendar__legend__label">TOIL PM Only</span>
+          </div>
+        </div>
+        <div class="col-sm-3">
+          <div class="chr_leave-calendar__legend__entry">
+            <div class="chr_leave-calendar__item chr_leave-calendar__item--requested chr_leave-calendar__item--type--toil"></div>
+            <span class="chr_leave-calendar__legend__label">Requested TOIL</span>
+          </div>
+        </div>
+        <div class="col-sm-3">
+          <div class="chr_leave-calendar__legend__entry">
+            <div class="chr_leave-calendar__item chr_leave-calendar__item--requested chr_leave-calendar__item--type--toil"><span>AM</span></div>
+            <span class="chr_leave-calendar__legend__label">Requested TOIL AM</span>
+          </div>
+          <div class="chr_leave-calendar__legend__entry">
+            <div class="chr_leave-calendar__item chr_leave-calendar__item--requested chr_leave-calendar__item--type--toil"><span>PM</span></div>
+            <span class="chr_leave-calendar__legend__label">Requested TOIL PM</span>
+          </div>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-sm-12">
+          <div class="chr_leave-calendar__legend__entry">
+            <div class="chr_leave-calendar__item chr_leave-calendar__item--type--toil chr_leave-calendar__item--alt"><span>AT</span></div>
+            <span class="chr_leave-calendar__legend__label">Accrued TOIL</span>
+          </div>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-sm-3">
+          <div class="chr_leave-calendar__legend__entry">
```

Comment:
I think its better to use a `Legend` directive which would accept `class` and `text`.